internal/pkg/cli/svc_init.go

```diff
@@ -51,6 +51,10 @@ const (
 	fmtAddSvcToAppStart    = "Creating ECR repositories for service %s."
 	fmtAddSvcToAppFailed   = "Failed to create ECR repositories for service %s.\n"
 	fmtAddSvcToAppComplete = "Created ECR repositories for service %s.\n"
+
+	wkldInitImagePrompt     = `What's the location of the image to use?`
+	wkldInitImagePromptHelp = `The location of an existing Docker image. Docker Hub registry are available by default.
```

Comment:
nit: I think here we can just say "name" instead of location so that we don't use the same term twice:
```suggestion
	wkldInitImagePromptHelp = `The name of an existing Docker image. 
Images in the Docker Hub registry are available by default.
```